src/js/base/BoxGroupSectionSelect.ts

```diff
@@ -1,17 +1,16 @@
 import styled, { DefaultTheme } from "styled-components";
-import { BoxGroupSection } from "./BoxGroupSection";
+import { BoxGroupSection, BoxGroupSectionProps } from "./BoxGroupSection";
 import { CheckboxLabel } from "./CheckboxLabel";
 import { StyledCheckbox } from "./styled/StyledCheckbox";
 
-type SelectBoxGroupSectionProps = {
+type SelectBoxGroupSectionProps = BoxGroupSectionProps & {
     active?: boolean;
     theme: DefaultTheme;
 };
 
 export const SelectBoxGroupSection = styled(BoxGroupSection)<SelectBoxGroupSectionProps>`
     background-color: ${props => (props.active ? props.theme.color.blue : "transparent")};
     color: ${props => (props.active ? props.theme.color.white : "inherit")};
-    cursor: pointer;
```

Comment:
Why this change?

Reply:
Initially made this change because `BoxGroupSection` already makes the cursor a pointer when there is an `onClick` and all of the `SelectBoxGroupSection` usages had an `onClick` I believe, so I found it redundant and since I changed the `onClick` to `Checkbox` I thought t shouldn't show a pointer for the whole item just the clickable area. I'll revert the changes made to this component since we don't use it in this PR anymore